changelog.txt

```diff
@@ -18,7 +18,7 @@ that repo.
 ## Fixes
 - `gui/gm-unit`: fixed behavior of ``+`` and ``-`` again to adjust skill values instead of populating the search field
 - `gui/create-item`: restrict materials to those normally allowed by the game by default, introduce new ``--unrestricted`` option for full freedom in choosing materials
-- `warn-stealers`: fixed script not working because of an incorrect event name
+- `warn-stealers`: fixed script not working because of an incorrect event name, storing unit pointers instead of IDs, and it not announcing creatures that spawn in revealed
```

Comment:
There can be multiple lines for the same tool if the changes are different. I think this meets that criteria. How about a new line that says:
```
- `warn-stealers`: properly announce thieving creatures that spawn already revealed
```